src/TUTORIAL.md

```diff
@@ -1152,8 +1158,13 @@ Now, if we `POST` something like this:
   "password": "hoistthesales!"
 }
 ```
+And `POST` it to the web-service;
+
+```shell
+$ curl -v -H "Content-Type: application/json" --data-binary @new-account.json http://localhost:3001/accounts
+```
 
-the record created will be something like this:
+...the record created will be something like this:
```

Comment:
Same problem with the ellipses as with the previous commit. It may be a bit confusing for the reader, as if we skipped something here, like some sort of a TBD.